app.rb

```diff
@@ -10,49 +10,90 @@
 
 # Use enumerables to -
 
+#warm-up
+#get the first apartment
+# puts apartment[0]
+# get square foot of first apartment
+# apartment[0][:square_feet][0]
+
   # Print all the addresses for the apartments
-apartments = each.do |address|
-  puts address [:apartments]
-end
+# apartments.each do |apartment|
+#   apartments[:address]
```

Comment:
This doesn't work because inside the enumerable you need to reference the variable you create with the pipes. You said `|apartment|` so in your block `apartment` is what you need to reference - so this should be `apartment[:address]` and to show it we would puts that statement. What you use for the enumerable variable (like all variable names) doesn't matter as long as you call it consistently. Ex.
```rb
apartments.each do |pizza|
  puts pizza[:address]
end
```